servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/ConnectionSelectorPolicies.java

```diff
@@ -18,20 +18,20 @@
 import io.servicetalk.client.api.LoadBalancedConnection;
 
 /**
- * A factory to create different {@link ConnectionPoolPolicy} variants.
+ * A factory to create different {@link ConnectionSelectorPolicy} variants.
  */
-public final class ConnectionPoolPolicies {
+public final class ConnectionSelectorPolicies {
     private static final int DEFAULT_MAX_EFFORT = 5;
     private static final int DEFAULT_LINEAR_SEARCH_SPACE = 16;
 
-    private ConnectionPoolPolicies() {
+    private ConnectionSelectorPolicies() {
         // no instances
     }
 
     /**
      * A connection selection policy that prioritizes a configurable "core" pool.
      * <p>
-     * This {@link ConnectionPoolPolicy} attempts to emulate the pooling behavior often seen in thread pools.
+     * This {@link ConnectionSelectorPolicy} attempts to emulate the pooling behavior often seen in thread pools.
      * Specifically it allows for the configuration of a "core pool" size which are intended to be long-lived.
      * Iteration starts in the core pool at a random position and then iterates through the entire core pool before
```

Comment:
Note that there are a bunch of usages of the term 'pool' in the documentation here. I think they make sense as many of them are used in analogy to a java thread pools notion of a 'core pool', but there is room for debate.